package/src/components/Attachment/AttachmentActions.tsx

```diff
@@ -79,7 +79,7 @@ const AttachmentActionsWithContext = <
           <TouchableOpacity
             key={`${index}-${action.value}`}
             onPress={() => {
-              if (action.name && action.value && handleAction) {
```

Comment:
whats the reason for this change?

Reply:
After taking the latest pull from the branch `develop` I was getting this warning:

```
This condition will always return true since this function is always defined. Did you mean to call it instead? ts(2774)
```

I tried removing the handler condition and things were working fine, so we don't need it. Let me know if I am missing something?